src/empire/city.h

```diff
@@ -4,6 +4,8 @@
 #include "core/buffer.h"
 #include "game/resource.h"
 
+#define MAX_TRADERS 3
```

Comment:
To avoid name collisions, defines in a header should have the file name, so in this case it should be `EMPIRE_CITY_MAX_TRADERS`.